guest-app/gulpfile.js

```diff
@@ -0,0 +1,11 @@
+var gulp = require('gulp');
+var webserver = require('gulp-webserver');
+
+gulp.task('guest-app-home', function () {
```

Comment:
Change task name to 'serve'. We use same for dashboard app, so we should stick to it.